hadoop-common-project/hadoop-common/pom.xml

```diff
@@ -334,6 +334,11 @@
       <artifactId>dnsjava</artifactId>
       <scope>compile</scope>
     </dependency>
+    <dependency>
+      <groupId>org.wildfly.openssl</groupId>
+      <artifactId>wildfly-openssl</artifactId>
+      <version>1.0.4.Final</version>
```

Comment:
1. move the declaration of the artifact & version intp `hadoop-project/pom.xml`
2. import it here and tag as <provided>.

That way it doesn't force a new JAR onto all downstream users (and the shading work to go with). The stores (abfs, s3a) will then have to explicitly add the wildfly dependency.

Reply:
done, marked the hadoop-common dependency as provided